### PR TITLE
RMF-369 API Gateway Release notes May 23 updates

### PR DESCRIPTION
**Pull request by andreamussap**
http://pr-2634.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230530_apimgr_relnotes/

## Changes

- Update Intro around PDF Analytics.
- Removed 3 notices from Deprecated section.
- Added Passport placeholder to Removed section (JBermi to provide the text)
- Removed 'Enable Unsafe Legacy Renegotiation on TLS Listeners' from Know Issue section

## Pending

- JBrennam to add sniffing feature to Removed section
- Know Issue Quotes in filter... Padraic will follow up

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | a9e69d7 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2634.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md [hunk 1]
```diff
@@ -70,30 +60,9 @@ There are no end of support notices in this update.
 
 To stay current and align our offerings with customer demand and best practices, Axway might discontinue support for some capabilities. As part of this update, the following features have been removed:
 
-### Sun Access Manager has been retired
```
Comment (JohnD-Axway) [inline content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:73]: @andreamussap Was this section in the last release?

Comment (andreamussap) [reply content/en/docs/apim_relnotes/20230530_apimgr_relnotes.md:73]: No. I thought it was, but no, it's new. I'll put it back. Thanks!

**Review by jmbermingham — APPROVED** [review]
Looks good, I'll send text for 'Passport' section.

**Pull request merged**